File: src/app-upload/abripourtous.js
function abripourtous()
{
 const user="abripourtous"
 const repo="abripourtous.github.io"
 const token=get_token(user)
 const url=concat("https://",token,"@github.com/",user,"/",repo,".git")
 const tmp=path_concat("tmp",repo)
 const index=path_concat(tmp,"index.html")
 const html=app_home("abripourtous")
 const www="/var/www/html"
 const robot="User-agent: *\nDisallow: /"

 flower(user)
 dir_make(tmp)
 
 fs_copy("src/app-abripourtous/res",tmp)
 file_save(index,html)
 
 os_system("sudo","rm","--recursive",www)
 os_system("sudo","mkdir",www)
 os_system("sudo","cp","--recursive",tmp,www)
 os_system("sudo","chmod","--recursive","777",www)
 file_save("/var/www/html/robots.txt",robot)
 
 git_upload(tmp,user,url)
}

File: src/app-upload/abripourtous.cs
gn abripourtous x:etc
 let user "abripourtous"
 let repo "abripourtous.github.io"
 let token app_token user
 let url git_url token user repo
 //let tmp path_concat "tmp" repo
 let tmp path_tmp
 let tmp path_concat tmp repo
 let index path_concat tmp "index.html"

 flower user

 let html app_home "abripourtous"

 dir_make tmp
 fs_copy "src/spa-abripourtous/res" tmp
 file_save index html
 www_copy tmp
 run git_upload tmp user url
end
